lockfiles/st2-constraints.txt

```diff
@@ -12,16 +12,16 @@
 # DROPS RESOLVED VERSION: <which version pip resolved without this constraint>
 #<package><version constraint>
 
+# REQUIRED BY: pytest-cov
+# REASON: 7.5 was causing errors with orquesta integration tests
+# NOTE: This was probably interaction w/ nose. Try to remove this after switch to pytest.
+# DROPS RESOLVED VERSION: 7.6.1
+coverage<7.5
```

Comment:
I moved this constraint from `test-requirements.txt` to here, as it is now part of the `lockfiles/st2.lock`. Then, I put the locked version back in `test-requirements.txt`.